boards/common/arduino-atmega/include/board_common.h

```diff
@@ -113,8 +113,12 @@ extern "C" {
 #define CONFIG_ZTIMER_USEC_DEV      (TIMER_DEV(0))
 #define CONFIG_ZTIMER_USEC_FREQ     (250000LU)
 #define CONFIG_ZTIMER_USEC_WIDTH    (16)
-#define CONFIG_ZTIMER_USEC_ADJUST_SET     (124)
+#ifndef CONFIG_ZTIMER_USEC_ADJUST_SET
+#define CONFIG_ZTIMER_USEC_ADJUST_SET     (132)
+#endif
+#ifndef CONFIG_ZTIMER_USEC_ADJUST_SLEEP
 #define CONFIG_ZTIMER_USEC_ADJUST_SLEEP   (120)
+#endif
```

Comment:
this should have the lower values  (never adjust by to much) by default 
```suggestion
#define CONFIG_ZTIMER_USEC_ADJUST_SET     (128)
#endif
#ifndef CONFIG_ZTIMER_USEC_ADJUST_SLEEP
#define CONFIG_ZTIMER_USEC_ADJUST_SLEEP   (116)
#endif
```

Reply:
addressed